### ISSUE TITLE
'keybase device add' -> error: argument "subcommand_name" invalid

### ISSUE THREAD

**Issue by auremoser** [keybase/client#1545]
### Context
I just contributed to an issue here [#1399](https://github.com/keybase/client/pull/1399) but as I suspect the following issue is distinct, I moved it to a new thread. I'm trying to add a new device to my keybase account with the `keybase device add` command listed [here in docs](https://keybase.io/docs/command_line). 

### Issue
On my new machine:
`keybase login`
returns a verification code and 
>If you are using the command line client on your other device, run this command:
>`keybase device add`

On my old machine:
`keybase device add` 
returns 
>keybase: error: argument "subcommand_name" Invalid choice: device...

and then proceeds to list a bunch of commands that are valid, but none relevant to adding devices

### What I tried
I thought maybe because I setup my old machine early, I would try uninstalling the node version and re-installing the brew package listed in docs. So I did that, no dice. Still the same issue :(.

Anyone else having problems adding devices, or have any tips about this, I'd be so happy to hear them.
Thanks!

**Comment by maxtaco** [comment]
Yeah, it seems like that's the old node version speaking on the old machine.

Can you run `keybase -d version` ? Thanks!

**Comment by auremoser** [comment]
Surely, yep, looks like uninstall via npm didn't work :( though it didn't through me any errors.
>debug:  Fallback to layout_v1
debug: + opening config file: /Users/aurelia/.keybase/config.json
debug: ++ loading config file /Users/aurelia/.keybase/config.json
debug: -- loaded config file -> {"user":{"name":"aure","salt":"4dee73e1afe527f8b90736cf281c6770","id":"6e4d139041933a393fc4cf683af48d00"}}
debug: - opened config file; found=true
debug: + testing GPG command-line client <default: gpg2 or gpg>
debug: | using GPG command: gpg2
debug: - tested GPG command-line client -> null
debug: Setting GPG_TTY=/dev/ttys001
debug: + Load proxy CAs
debug: - Loaded proxy CAs
debug: + startup message
debug: | CLI version: 0.7.5
debug: | Platform info: {"versions":{"http_parser":"1.0","node":"0.10.22","v8":"3.14.5.9","ares":"1.9.0-DEV","uv":"0.10.19","zlib":"1.2.3","modules":"11","openssl":"1.0.1e"},"arch":"x64","platform":"darwin","features":{"debug":false,"uv":true,"ipv6":true,"tls_npn":true,"tls_sni":true,"tls":true}}
debug: | Version info: ["keybase (keybase.io CLI) v0.7.5","- node.js v0.10.22","- gpg (GnuPG/MacGPG2) 2.0.22","- libgcrypt 1.5.3","Identifies as: 'keybase.io node.js client v0.7.5 darwin'"]
debug: - startup message
debug: + opening database file: /Users/aurelia/.keybase/keybase.idb
debug: ++ DB::_init_db
debug: -- DB::_init_db -> 0
debug: - DB opened
debug:  Call to gpg: { args: [ '--version' ] }
keybase (keybase.io CLI) v0.7.5
- node.js v0.10.22
- gpg (GnuPG/MacGPG2) 2.0.22
- libgcrypt 1.5.3
Identifies as: 'keybase.io node.js client v0.7.5 darwin'

**Comment by maxtaco** [comment]
Well that's sad. It is an old version of node and npm, so it might be buggy. Is it possible you did an `npm install` as `root`, and it didn't have permissions to uninstall without privileges?

Thanks, and sorry for these snafus!
